src/app/@theme/components/header/header.component.ts

```diff
@@ -91,4 +93,10 @@ export class HeaderComponent implements OnInit, OnDestroy {
     this.menuService.navigateHome();
     return false;
   }
+
+  contextItemClick(title) {
+    if (title === "Log out") {
```

Comment:
Isn't it possible to compare by id, enum or anything else? For example: if (item === MenuItem.LogOut)